engine/runtime/src/test/java/org/enso/interpreter/test/TypeMembersTest.java

```diff
@@ -87,6 +88,21 @@ public void checkAtomMembers() throws Exception {
     assertMembers("Keys in list2", true, endAtom, "h", "t");
   }
 
+  @Test
+  public void ensureNonBuiltinMembersArePresent() throws Exception {
+    final URI uri = new URI("memory://how_long.enso");
+    final Source src = Source.newBuilder("enso", """
+    from Standard.Base.Errors.Common import Compile_Error
```

Comment:
You might also ask why. Well, it's because the `definitionScope` of the constructor is in builtins module, while in `Cmon.Error` it is in that `Cmon` one. Builtins are... special.

Reply:
If I forced you to write a "unit" test for something that can't be unit tested, then I deeply apologize. But to not give up so easily. Would:

```
@Builtin_Type
type Compile_Error
    Error message
```
rather than `from Standard.Base.Errors.Common import Compile_Error` in the test work? But it'd be cheating and not a unit test anyway...

Reply:
No worries. I will check that suggestion